### DISCUSSION TITLE
ol is not defined

### DISCUSSION THREAD

**Discussion by zeMinng** [openlayers/openlayers#15714] [Q&A] [answered]
ol is not defined

of "ol": "7.1.0",

**Comment by ahocevar** [comment]
What were you trying to do? Where were you looking for `ol`? Did you do something like
```js
import { Map, View } from 'ol';

const map = new Map({
  view: new View({
    center: [0, 0],
    zoom: 2
  })
});
```
?

**Reply by zeMinng** [reply]
First Steps：npm i ol@7.1.0

next: 
 new ol.layer.Tile({
      maxZoom: 18,
      source: new ol.source.TileImage({
        url: 'http://10.78.164.100:8000/googlemaps/satellite_en/{z}/{x}/{y}.jpg'
      })
    }),
    
  Ask for ol is not defined in the browser
  Looking forward to your answer

**Comment by zeMinng** [comment]
I already understand this is
![image](https://github.com/openlayers/openlayers/assets/75962652/eda24cbf-42f2-426f-a3b0-a8be4ebe7692)
![image](https://github.com/openlayers/openlayers/assets/75962652/71f7f065-9e80-4cf5-ab5a-fe61722511c3)

but，I don't understand why the previous project was written differently
![image](https://github.com/openlayers/openlayers/assets/75962652/ae2d7b04-7205-43c4-b271-44f46218f5d6)

If I continue to use ol.layer.Tile, I will make an error: ol is not defined

**Comment by ahocevar** [comment, marked as answer]
If you want to use the `ol.layer.Tile` syntax, you need to use the legacy build, which you find in the package as well (`./dist/ol.js`).

**Reply by zeMinng** [reply]
ok, thank you
